MAUI/DataForm/layout.md

```diff
@@ -0,0 +1,372 @@
+---
+layout: post
+title: Layout in .NET MAUI DataForm control | Syncfusion
+description: Learn here all about the Layout support in Syncfusion .NET MAUI DataForm (SfDataForm) control and more.
```

Comment:
Description: Learn about the Layout support in Syncfusion .NET MAUI DataForm (SfDataForm) control and more.

Reply:
Content changed